opty/direct_collocation.py

```diff
@@ -197,6 +197,54 @@ def __init__(self, obj, obj_grad, equations_of_motion, state_symbols,
 
         self.obj_value = []
 
+    # this is added only to correct the docstring, which is a bit different with
+    # opty compared to cyipopt.
```

Comment:
These two lines are not need. If is common to overwrite methods in sub classes for this reason and it needs no explanation.